forc/src/cli/plugin.rs

```diff
@@ -14,14 +14,35 @@ use std::{
 ///
 /// E.g. given `foo bar baz` where `foo` is an unrecognized subcommand to `forc`, tries to execute
 /// `forc-foo bar baz`.
-pub(crate) fn execute_external_subcommand(args: Vec<String>) -> Result<process::Output> {
+pub(crate) fn execute_external_subcommand(
+    args: Vec<String>,
+    is_verbose: bool,
+) -> Result<process::Output> {
     let cmd = args.get(0).expect("`args` must not be empty");
     let args = &args[1..];
     let path = find_external_subcommand(cmd);
     let command = match path {
         Some(command) => command,
         None => bail!("no such subcommand: `{}`", cmd),
     };
+
+    if let Ok(forc_path) = std::env::current_exe() {
+        if is_verbose && command.parent() != forc_path.parent() {
```

Comment:
I think we should always show this warning, not only in verbose mode

Reply:
This was Cesar's original plan but i asked for it to be behind a verbose flag. Mainly so it doesn't get annoying for people that intentionally have builds in multiple places like we do. 

I guess considering we are the minority, perhaps we could take another option by using a silent flag instead to not show warnings, but have them show by default?